packages/dso-toolkit/src/components/rich-content/rich-content.mixins.scss

```diff
@@ -15,6 +15,11 @@
     &:not(.list-group):not(.dso-link-list):not(.dso-columns-list) {
       padding-inline-start: units.$u3;
     }
+
+    ul,
+    ol {
+      margin-block-end: 0;
```

Comment:
Waar etaleren we dit in Storybook?

Reply:
http://localhost:45000/?path=/story/voorbeeldpagina-s-toepassingen-aanvragen-locatie--locatie

Reply:
En nu wel 😎

Reply:
En de build faalt nu. En dat is precies wat er verwacht mag worden, omdat de matchImageScreenshot nu afwijkt van de baseline door de toevoeging van de de geneste ul en ol in `rich-content.content.ts`